templates/email/bexley/waste/_sidebar_content.html

```diff
@@ -1,16 +1,14 @@
 [% # Hacky string munging :-( %]
+[% containers = cobrand.container_desc_for_template %]
 [%
   address = report.get_extra_metadata.property_address;
   IF is_no_collections_enquiry;
     title = report.detail;
     title = title.remove(address);
-  ELSIF is_missed_collection;
-    title = report.title;
-    title = title.remove('Report missed ');
   END
 %]
 <h2 style="[% h2_style %]">Bins reported as missed:</h2>
-[% title | html_para_email(secondary_p_style) %]
+[% report.title | html_para_email(secondary_p_style) %]
```

Comment:
This line still needs to be title, otherwise the 'is_no_collections_enquiry" changes are lost.